All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+
+**Jaye Harrill**
+That makes sense.
+
+**Pooja Ranjan**
+So I remember an incident when I submitted a PR for a specific EIP, which was already existing. And then one of the editors edited my EIP number. I forgot to mention the hyphen between, say, for example, it's EIP 1, so I chose the third one: EIP-space-1, and he corrected it mentioning that no, it needs to be EIP-hyphen-1. So if we consider that as a baseline, I think that needs to be mentioned in EIP 1, and that can be followed.
+
+**Jaye Harrill**
+Yeah, I had another question. I'm just looking at the Constantinople post mortem action items. Emergency Comms Team and also Risk Evaluation Rubric were two things that popped up. The thing about the comms team - is there any progress on that? Because I feel like there's a few people in this space would be perfect for that. And certainly, we have some experience with doing emergency comms. And then the risk evaluation, this little note is small, and what is involved in creating that is very, very large. I think it would be better to put that up for opinion in the Eth security chat.
+
+**James Hancock**
+For the naming conventions?
+
+**Jaye Harrill**
+"Create a rubric for risk evaluation."
+
+**James Hancock**
+Oh, yes. Yes.
+
+**Jaye Harrill**
+Yes. I don't know that that's - you can have something to base things off of, but that is not going to necessarily cover all your bases. And I know don't know that there's necessarily a way to cover all your bases, but there are certainly things that are doable.
+
+**James Hancock**
+Yeah, that would be great to get the list of things. I don't know if there's any progress on a rubric for comms. It ends up being Hudson and I waking up whenever something happens and finding people that need to talk about it. I'm pinging him and pinging everyone until things are resolved.
+
+**Jaye Harrill**
+Is there a playbook as to what the checks are to do when that happens, all the checks that need to be need to be done?
+
+**James Hancock**
+Not that I'm aware of.
+
+**Pooja Ranjan**
+So for that, we have created this emergency comm process document, sometimes back during the Constantinople post mortem only. So it suggests that we should be having a list of key audiences, those are going to be, you know, emergency comms members. And we are trying to create this list. Generally, we use this kind of list at the time of upgrade or hard fork to communicate with our key stakeholders to let them know what's going on if something urgent comes up. So that's it, and the other thing was we came up with a list of preferred channels of communication, because it's important that we should be reaching people, more people and as quickly as possible. So these are the preferred, appropriate channels that we suggested we should be using.
```

Comment:
So for that, we have created this [emergency comm process document](https://github.com/ethereum-cat-herders/PM/blob/63d6c017c67ac2188b44f8d9f5293f563a1745cf/Constantinople%20Postmortem%20Report/EmergencyCommTemplate.md) during the Constantinople post mortem. So it suggests that we should be having a list of key audiences, those are going to be emergency comms members. And we are trying to create this list. Generally, we use this kind of list at the time of upgrade or hard fork to communicate with our key stakeholders to let them know what's going on if something urgent comes up. So that's it, and the other thing was we came up with a list of [preferred channels of communication](https://github.com/ethereum-cat-herders/PM/blob/63d6c017c67ac2188b44f8d9f5293f563a1745cf/Constantinople%20Postmortem%20Report/EmergencyCommTemplate.md#preferred--appropriate-channels-of-communication) because it's important that we should be reaching people, more people and as quickly as possible. So these are the preferred, appropriate channels that we suggested we should be using.